api/users/permissions.py

```diff
@@ -0,0 +1,15 @@
+from website.models import User
+from rest_framework import permissions
+
+
+class ReadOnlyOrCurrentUser(permissions.BasePermission):
+    # Checks to see if the request is coming from the currently logged in user, and allows
```

Comment:
This can be the docstring for the class. As per PEP 257, use the imperative mode, e.g. "Check to see if the request..." rather than "Checks to see..."